qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -363,11 +405,43 @@ <h5>
 <div class="row">
   <div class="col-md-12">
     <ul class="nav nav-pills">
+      <li style="border: 1px solid #428bca; border-radius: 5px"><a data-toggle="tab" href="#sample-listing-tab-div">Sample Listing</a></li>
       <li style="border: 1px solid #428bca; border-radius: 5px"><a data-toggle="tab" href="#summary-tab-div">Summary</a></li>
       <li style="border: 1px solid #428bca; border-radius: 5px" class="active"><a data-toggle="tab" href="#processing-graph-vue" id="processing-tab-btn">Processing</a></li>
     </ul>
 
     <div class="tab-content">
+      <div id="sample-listing-tab-div" class="tab-pane fade">
+        <div class="panel panel-default">
+          {% if not artifact_attached %}
+            <button class="btn btn-danger st-interactive" onclick="delete_prep_sample({{prep_id}});">
+                <span class="glyphicon glyphicon-trash"></span> Delete Selected
+            </button>
+          {% end %}
+          <div>
+            <table class="table">
+              <thead>
+                <tr>
+                  {% if not artifact_attached %}
+                    <th><input type="checkbox" onchange="toggleCheckboxes(this)"></th>
+                  {% end %}
+                  <th>Samples</th>
+                </tr>
+              </thead>
+              {% for sid in samples %}
+                <tr id="row">
```

Comment:
Isn't this id going to be repeated for all elements?

Reply:
ha! good catch!

Reply:
note that this is fine cause we want a new row for each sample ...

Reply:
I see, the problem is still that the attribute is included and it is not used anywhere in the code and has a repeated value (for each row) so it wouldn't be usable anyways. Can you remove it?